.github/workflows/publish_docker.yml

```diff
@@ -4,8 +4,8 @@ on:
   workflow_dispatch:
     inputs:
       vm_id:
-        description: 'The ID of the VM (binary dst in Docker image)'
-        default: ''
+        description: "The ID of the VM (binary dst in Docker image)"
+        default: ""
         required: false
         type: string
```

Comment:
also - maybe out of scope - but shouldn't we publish docker images on releases as well? 🤔

Reply:
we do release on tag pushes. I guess there is also "release" specific trigger, but I think tag trigger includes release as well.

Reply:
My apologies, the on tag trigger was collapsed in the diffs - never mind completely!